### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by corny**
this replaces #183

### CODE REVIEW

**Comment by corny** [comment]
@czerwonk Ping

File: cmd/ris/risserver/server.go [hunk 1]
```diff
@@ -10,6 +10,7 @@ import (
 	"github.com/bio-routing/bio-rd/routingtable"
 	"github.com/bio-routing/bio-rd/routingtable/filter"
 	"github.com/bio-routing/bio-rd/routingtable/locRIB"
+	"github.com/pkg/errors"
```
Comment (q3k) [inline cmd/ris/risserver/server.go]: I would wait for Go2 errors, or replace `github.com/pkg/error` with something Go2-error-proposal compatible.

Comment (corny) [reply cmd/ris/risserver/server.go]: Do you know a package that is closer to Go2-error-proposal?

Comment (q3k) [reply cmd/ris/risserver/server.go]: https://godoc.org/golang.org/x/xerrors, which also claims to be partially merged into 1.13.

**Review by hikhvar — CHANGES_REQUESTED** [review]
Please use fmt.Errorf with %w instead of github.com/pkg/errors

**Pull request closed without merging**